### PR TITLE
[WIP]Swift entities generator

### PR DESCRIPTION
**Pull request by PaulTaykalo**

### CODE REVIEW

File: codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/SwiftGenerationOptions.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.stanfy.helium.handler.codegen.swift.entity
+
+import com.stanfy.helium.handler.codegen.GeneratorOptions
+
+/**
+ * Created by paultaykalo on 6/25/16.
```
Comment (roman-mazur) [inline codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/SwiftGenerationOptions.kt]: Remove

File: codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/SwiftEntitiesGenerator.kt [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.stanfy.helium.handler.codegen.swift.entity
+
+import com.stanfy.helium.handler.Handler
+import com.stanfy.helium.handler.codegen.BaseGenerator
+import com.stanfy.helium.handler.codegen.swift.entity.filegenerator.SwiftFilesGenerator
+import com.stanfy.helium.model.Project
+import java.io.File
+
+/**
+ * Created by paultaykalo on 6/25/16.
```
Comment (roman-mazur) [inline codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/SwiftEntitiesGenerator.kt]: Remove

Comment (roman-mazur) [inline codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/SwiftEntitiesGenerator.kt]: Or put relevant docs

File: codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/filegenerator/SwiftFile.kt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package com.stanfy.helium.handler.codegen.swift.entity.filegenerator
+
+/**
+ * Created by paultaykalo on 6/25/16.
```
Comment (roman-mazur) [inline codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/filegenerator/SwiftFile.kt]: Remove

File: codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/filegenerator/SwiftFilesGenerator.kt [hunk 4]
```diff
@@ -0,0 +1,32 @@
+package com.stanfy.helium.handler.codegen.swift.entity.filegenerator
+
+import org.apache.commons.io.IOUtils
+import java.io.File
+import java.io.FileOutputStream
+import java.io.OutputStreamWriter
+import java.io.Writer
+
+/**
+ * Created by paultaykalo on 6/25/16.
```
Comment (roman-mazur) [inline codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/filegenerator/SwiftFilesGenerator.kt]: Remove

File: codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/filegenerator/SwiftFilesGenerator.kt [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+          swiftFile.contents())
+    }
+
+  }
+  private fun writeFile(file: File, contents: String) {
+    var output: Writer? = null
+    try {
+      output = OutputStreamWriter(FileOutputStream(file), "UTF-8");
+      output.write(contents);
+    } catch (e: RuntimeException) {
```
Comment (roman-mazur) [inline codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/filegenerator/SwiftFilesGenerator.kt]: IOException

**Comment by roman-mazur** [comment]
So do you feel we can merge this?

**Comment by PaulTaykalo** [comment]
@roman-mazur Yep, this one is fine, I think

**Pull request merged**